eos/rare-b-decays/b-to-kstar-ll-gvdv2020_TEST.cc

```diff
@@ -255,3 +255,199 @@ class BToKstarDileptonJavierTest :
+            TEST_CHECK_RELATIVE_ERROR(real(nff->H_perp(q2)),     0.000115032,   eps);
+            TEST_CHECK_RELATIVE_ERROR(imag(nff->H_perp(q2)),     8.14984e-06,   eps);
+            TEST_CHECK_RELATIVE_ERROR(real(nff->Hhat_para(q2)),  0.0001583608,  eps);
+            TEST_CHECK_RELATIVE_ERROR(imag(nff->Hhat_para(q2)),  1.205929e-05,  eps);
+            TEST_CHECK_RELATIVE_ERROR(real(nff->H_para(q2)),     0.000114113,   eps);
+            TEST_CHECK_RELATIVE_ERROR(imag(nff->H_para(q2)),     8.68978e-06,   eps);
+            TEST_CHECK_RELATIVE_ERROR(real(nff->Hhat_long(q2)), -3.68996e-06,   eps);
+            TEST_CHECK_RELATIVE_ERROR(imag(nff->Hhat_long(q2)),  2.74982e-07,   eps);
+            TEST_CHECK_RELATIVE_ERROR(real(nff->H_long(q2)),     6.89722e-08,   eps);
+            TEST_CHECK_RELATIVE_ERROR(imag(nff->H_long(q2)),    -5.13967e-09,   eps);
+
+
+            Kinematics k
+            {
+                {"q2", q2},
+            };
+
+            // Javier's results for c7 = 1.0
+            // J's = { 9.88734e-21, 1.60842e-20,
+            // 3.11517e-21, -1.4707e-20, -2.6687e-22,
+            // 6.23113e-21, -1.12587e-20, -6.47412e-21, 0,
+            // 1.00589e-22, -5.21581e-23, -1.72649e-24 }
+
+            BToKstarDilepton c(p, oo);
+
+            TEST_CHECK_RELATIVE_ERROR(c.differential_j_1s(q2),  3.0/4.0*9.88734e-21,  eps);
+            TEST_CHECK_RELATIVE_ERROR(c.differential_j_1c(q2),  3.0/4.0*1.60842e-20,  eps);
+            TEST_CHECK_RELATIVE_ERROR(c.differential_j_2s(q2),  3.0/4.0*3.11517e-21,  eps);
+            TEST_CHECK_RELATIVE_ERROR(c.differential_j_2c(q2), -3.0/4.0*1.4707e-20,  eps);
+            TEST_CHECK_RELATIVE_ERROR(c.differential_j_3(q2),  -3.0/4.0*2.6687e-22,  eps);
+            TEST_CHECK_RELATIVE_ERROR(c.differential_j_4(q2),   3.0/4.0*6.23113e-21,  eps);
+            TEST_CHECK_RELATIVE_ERROR(c.differential_j_5(q2),  -3.0/4.0*1.12587e-20,  eps);
+            TEST_CHECK_RELATIVE_ERROR(c.differential_j_6s(q2), -3.0/4.0*6.47412e-21,  eps);
+            TEST_CHECK_NEARLY_EQUAL(c.differential_j_6c(q2),  0.0,  1e-22);
+            TEST_CHECK_RELATIVE_ERROR(c.differential_j_7(q2),   3.0/4.0*1.00589e-22,  eps);
+            TEST_CHECK_RELATIVE_ERROR(c.differential_j_8(q2),  -3.0/4.0*5.21581e-23,  eps);
+            TEST_CHECK_RELATIVE_ERROR(c.differential_j_9(q2),  -3.0/4.0*1.72649e-24,  eps);
+
+
+            // Javier's results
+            // {BR,FL,AFB,P1,P2,P3,P4p,P5p,P6p,P8p,S3,S4,S5,S7,S8,S9} =
+            // {0.669443, 0.506906, 0.167357, -0.0428339, -0.259782, 0.000138555, 0.920583, -0.831674, -0.00743048, 0.0077058, -0.00919817, 0.214768, -0.388051, 0.00346699, -0.00179773, -0.0000595069}  
+
+            TEST_CHECK_RELATIVE_ERROR(
+                 Observable::make("B->K^*ll::dBR/ds", p, k, oo)->evaluate(),    0.669443e-07, eps );
+            TEST_CHECK_RELATIVE_ERROR(
+                 Observable::make("B->K^*ll::F_L(q2)", p, k, oo)->evaluate(),   0.542507,     eps );
```

Comment:
The discrepancy to Javier's results in F_L requires a comment. If typo in Javier's results (as discussed), please update the comment above.

Reply:
There was indeed a typo in Javier's code, I found the discussion, I corrected the value but not the comment!